src/compas/datastructures/network/core/graph.py

```diff
@@ -1393,9 +1427,9 @@ def has_edge(self, u, v, directed=True):
         u : hashable
             The identifier of the first node of the edge.
         v : hashable
-            The identifier of the secondt node of the edge.
+            The identifier of the second node of the edge.
         directed : bool, optional
-            Take into accoun the direction of the edge.
+            Take into account the direction of the edge.
```

Comment:
Those damn t's do tend to wander around.  Have to keep them in line!

Reply:
😛